.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve Streamlink
+
+---
+
+<!--
+Thanks for reporting an issue!
+
+Please read the contribution guidelines (https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink) first!
```

Comment:
I think it would be better to have the (long) URL on the next line, to not break the sentence apart (readability).
Same issue also in the other templates...